deployment/ansible/group_vars/development_template

```diff
@@ -25,3 +25,11 @@ use_s3_storage: false
 default_admin_username: 'admin'
 default_admin_password: 'admin'
 default_admin_email: 'systems+cac@azavea.com'
+
+postgresql_version: "9.4"
```

Comment:
These PostgreSQL overrides aren't being seen by CI because `VAGRANT_ENV=TEST` exists in the Jenkins configuration. This makes the Vagrant configuration for Ansible use the `group_vars/test` (in addition to `all`) file for variables override, but not `development`.

https://github.com/azavea/cac-tripplanner/blob/1f9751837917c0bc344307cf74231445fe00e4da/Vagrantfile#L81-L85